build.gradle

```diff
@@ -366,19 +361,6 @@ task setupGeneratedDir(type: Copy) {
 }
 tasks.jib.dependsOn setupGeneratedDir
 
-liquibase {
```

Comment:
Not sure what the implications of removing this plugin config are. I assume this is separate from the change to use the library for the java code?

Reply:
It removes the liquibase gradle targets. I don't think we are using them now that we do the migration in code.